packages/mongo-nav/src/helpers/Icons.tsx

```diff
@@ -6,6 +6,171 @@ type ProductIconProps = HTMLElementProps<'svg'> & {
   active: boolean;
 };
 
+export function CloudManagerSmall() {
+  return (
+    <svg width="10px" height="10px" viewBox="0 0 10 10">
+      <path
+        className={css`
+          fill: #00804b;
```

Comment:
Can these just be direct attributes and remove emotion from the equation?

Reply:
👍